compat/mangle.json

```diff
@@ -20,6 +20,7 @@
       "$_root": "__p",
       "$_commit": "__c",
       "$_suspensions": "__u",
+      "$_fallback": "__",
```

Comment:
https://github.com/preactjs/preact/blob/b051204ca2dbcd409288a6a74d1ec8fbeae94fd9/compat/src/suspense.js#L46

Reply:
@JoviDeCroock 
Thank you for review.
I am sorry.
I changed this line for #2044.